tests/Query/HostsFileExecutorTest.php

```diff
@@ -18,8 +20,8 @@ class HostsFileExecutorTest extends TestCase
      */
     public function setUpMocks()
     {
-        $this->hosts = $this->getMockBuilder('React\Dns\Config\HostsFile')->disableOriginalConstructor()->getMock();
-        $this->fallback = $this->getMockBuilder('React\Dns\Query\ExecutorInterface')->getMock();
+        $this->hosts = $this->getMockBuilder(HostsFile::class)->disableOriginalConstructor()->getMock();
```

Comment:
Not sure, but can this be replaced like this?

```suggestion
        $this->hosts = $this->createMock(HostsFile::class);
```

Reply:
Yeah, I tried this before with the `disableOriginalConstructor()` still in place and it made the tests fail, but seems to work your way :+1: